<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>startscreen</class>
 <widget class="QDialog" name="startscreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>29</y>
     <width>351</width>
     <height>261</height>
    </rect>
   </property>
   <widget class="LoginForm" name="loginWidget"/>
   <widget class="RegistrationForm" name="registrationWidget"/>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LoginForm</class>
   <extends>QWidget</extends>
   <header>loginform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RegistrationForm</class>
   <extends>QWidget</extends>
   <header>registrationform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
